Sources/TuistGenerator/Generator/ProjectGenerator.swift

```diff
@@ -119,6 +117,9 @@ final class ProjectGenerator: ProjectGenerating {
                                     configurationList: XCConfigurationList,
                                     groups: ProjectGroups,
                                     pbxproj: PBXProj) throws -> PBXProject {
+        // sort products
+        groups.products.children.sort(by: { (p1, p2) in p1.nameOrPath < p2.nameOrPath })
```

Comment:
I'd rather sort them before they get added to the Xcode project.

Reply:
Totally agree, but I think that's what we do here, the products are assigned to the `PBXProject` few lines lower. It probably would be nice to have a separated method taking care or the last minute ordering.. in such case, maybe.